codalab/lib/server_util.py

```diff
@@ -166,11 +166,19 @@ def bottle_patch(path=None, **options):
     return app().route(path, 'PATCH', **options)
 
 
+def safe_uri(redirect_uri):
+    """Check if an URI is relative, otherwise raise an error.
+    """
+    if '://' in redirect_uri or '//' in redirect_uri:
```

Comment:
Let's use this method instead: https://stackoverflow.com/questions/8357098/how-can-i-check-if-a-url-is-absolute-using-python/8357518

Reply:
Updated, thanks!